scripts/prettier.js

```diff
@@ -55,4 +28,67 @@ async function runPrettierForAll() {
   });
 }
 
-runPrettierForAll();
+function parseArgs() {
+  return require('yargs')
+    .usage('Usage: prettier [commitHash] [options]')
+    .example('prettier', 'run prettier only on changed files')
+    .example('prettier HEAD~3', 'run prettier only on changed files since HEAD~3')
+    .options({
+      all: {
+        description: 'Run prettier on all projects',
```

Comment:
It's probably more accurate to say "all files" (and might also not hurt to add a note that it runs on files even if they didn't change)
```suggestion
        description: 'Run prettier on all files',
```

Reply:
good idea. I'd suggest to redo this once there is some common terminology in place etc.